src/content/configuration/other-options.mdx

```diff
@@ -66,6 +67,8 @@ module.exports = {
 
 This will force webpack to exit its bundling process.
 
+W> You can use `bail` option in [`watch`](/configuration/watch) mode, but it will still force webpack to exit as soon as an error is found.
```

Comment:
```suggestion
W> Avoid using `bail` option in [`watch`](/configuration/watch) mode, as it will force webpack to exit as soon as possible when an error is found.
```